### DISCUSSION TITLE
[Typo] There is a typo in README.md

### DISCUSSION THREAD

**Discussion by Bistard** [lowlighter/metrics#1336] [General]
In the file `https://github.com/lowlighter/metrics/blob/master/source/plugins/languages/README.md`, I believe the following is a typo:
![image](https://user-images.githubusercontent.com/38385498/210192918-66fa9217-7502-492d-b8a0-14e691a92fbf.png)
It supposed to be `plugin_repositories_skipped`.

**Comment by lowlighter** [comment]
Hi 👋 !

There is indeed an options called [`repositories_skipped`](https://github.com/lowlighter/metrics/blob/master/source/plugins/base/README.md#repositories_skipped), it is the global one 😅
